e2e-test/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM cypress/base:18.12.1
+FROM cypress/base:latest
```

Comment:
Please pick a fixed version that we know is stable, latest may not always be stable